lib/ex_aws/cloudformation.ex

```diff
@@ -40,23 +40,13 @@ defmodule ExAws.Cloudformation do
 
   @version "2010-05-15"
 
-  @type accepted_capability_vals ::
```

Comment:
I don't see how the type spec for this helps for the following reasons:
1) When AWS makes changes, we would need to update the typespec to reflect those changes. 
2) Easier for the user to pass in the correct value that AWS dictates in their docs instead of counting on some kind of helper function to convert the atom of the accepted values. One thing I noticed about AWS is that the values allowed can be very inconsistent. Safer for people using this library to just pass in the correct value.